packages/server/data/schema/scoreboard.json

```diff
@@ -119,7 +119,13 @@
         "round": { "type": "string" },
         "tournamentName": { "type": "string" },
         "caster": { "type": "string" },
-        "streamer": { "type": "string" }
+        "streamer": { "type": "string" },
```

Comment:
Nice! You haven't forgotten about the schema!

Have a look on issue #6, for ideas on how to better use overlays, btw :)